test/autograd/test_functional.py

```diff
@@ -8,7 +9,45 @@
 
 from torch.testing._internal.common_cuda import TEST_CUDA
 from torch.testing._internal.common_utils import (
-    TestCase, run_tests, gradcheck, gradgradcheck, parametrize, instantiate_parametrized_tests)
+    TestCase, run_tests, subtest, gradcheck, gradgradcheck, parametrize, instantiate_parametrized_tests)
+from torch.testing._internal.logging_tensor import LoggingTensor
+
+# Utilities for parametrizing the tensor constructors used in autograd tests
+base_ctors_dict = {
+    "ones": torch.ones,
+    "zeros": torch.zeros,
+    "randn": torch.randn,
+    "rand": torch.rand,
+    "tensor": torch.tensor,
+    # TODO: maybe move somewhere so other tests can also use
+    # TODO: according to https://pytorch.org/cppdocs/notes/tensor_creation.html
```

Comment:
Do we need these other functions? Is that really a TODO?

Reply:
I'm fine with adding them as needed. And it is not really a TODO to be exhaustive here.